cmd/controller-manager/app/options/options.go

```diff
@@ -228,7 +234,7 @@ func (o *GenericControllerManagerOptions) Validate() []error {
 	errors = append(errors, o.InsecureServing.Validate()...)
 	errors = append(errors, o.Authentication.Validate()...)
 	errors = append(errors, o.Authorization.Validate()...)
-
+	errors = append(errors, o.Debugging.Validate()...)
```

Comment:
nit: the order of calls should always be the same. Here Debugging is the last. For ApplyTo it is the first.